segmentation/utils/spatialanalysis_utils_test.py

```diff
@@ -1,12 +0,0 @@
-import numpy as np
-from segmentation.utils import spatialanalysis_utils
-
-
```

Comment:
We're going to be putting more stuff in here in the next round, don't delete this yet